lms/static/styles/_frontend-shared.scss

```diff
@@ -8,13 +8,14 @@
   // The containing `Modal` component will take up 90vw on narrow screens.
   // For viewports that are wider, `Modal` sets a min-width, but it's a little
   // narrow for several LMS Dialog interfaces.
+  @media screen and (min-width: 48rem) {
```

Comment:
This `48rem` comes from the breakpoints that the [`modal` mixin](https://github.com/hypothesis/frontend-shared/blob/9ca807b469b3b21743e64fbe3a7f5c6da9d19cde/styles/mixins/patterns/_containers.scss#L116). It would be good to make that connection a little more explicit.